src/main/java/org/hydev/mcpm/client/commands/entries/PageController.java

```diff
@@ -0,0 +1,98 @@
+package org.hydev.mcpm.client.commands.entries;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.apache.commons.lang3.StringUtils;
+import org.hydev.mcpm.Constants;
+import org.hydev.mcpm.client.commands.presenter.PagedPresenter;
+
+import javax.annotation.Nullable;
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * Controller for the page command
+ *
+ * @author Azalea (https://github.com/hykilpikonna)
+ * @since 2022-11-24
+ */
+public class PageController implements PageBoundary
+{
+    private static final File PERSIST_PATH = new File(Constants.CFG_PATH, "page-info.json");
```

Comment:
I believe a gateway interface is required here for adherence to clean architecture.

Reply:
Maybe put the `StringStorage` interface in `feat-export-import` to use?